src/ORM/Queries/SQLConditionalExpression.php

```diff
@@ -287,6 +287,13 @@ public function getJoins(&$parameters = array())
                     continue;
                 }
 
+                if (preg_match('/AS\s+(:?"[^"]+"|[A-Z0-9_]+)\s*$/i', $join)) {
```

Comment:
Should `:?` go outside of that capturing group? Or should it be `("[^"]+"|:?[A-Z0-9_]+)`?

Reply:
Or should this be a non capturing group `(?: ... )`

Reply:
whoops, that actually should be `?:` :)

Reply:
patched